specs/commandline/inspect.md

```diff
@@ -0,0 +1,271 @@
+# notation inspect
+
+## Description
+
+Use `notation inspect` command to inspect all the signatures associated with signed artifact in a human readable format.
+
+Upon successful execution, both the digest of the signed artifact and the digests of signatures manifest along with their properties associated with the signed artifact are printed in the following format:
+
+```shell
+<registry>/<repository>@<digest>
+└──application/vnd.cncf.notary.signature
+  ├──<digest_of_signature_manifest>
+    ├──<signingAlgorithm>
+    ├──<signedAttributes>
+    ├──<userDefinedAttributes>
+    ├──<unsignedAttributes>
+    ├──<certificates>
+    └──<payload>
+  ├──<digest_of_signature_manifest>
+    ├──<signingAlgorithm>
+    ├──<signedAttributes>
+    ├──<unsignedAttributes>
+    ├──<certificates>
+    └──<payload>
+```
+
+## Outline
+
+```text
+Inspect all signatures with the signed artifact.
+
+Usage:
+    notation inspect [flags] <reference>
+  
+Flags:
+   -h, --help              help for describing the signature
+   -o, --output json       output on command line sets the output to json
+   -p, --password string   password for registry operations (default to $NOTATION_PASSWORD if not specified)
+       --plain-http        registry access via plain HTTP
+   -u, --username string   username for registry operations (default to $NOTATION_USERNAME if not specified)
+```
+
+## Usage
+
+### Display the details of all the listed signatures and its associated certificate properties of the signed container image
+
+
+```text
+notation inspect [flags] <registry>/<repository>@<digest>
+```
+
+## Inspect signatures on the supplied OCI artifact identified by the digest
+
+```shell
+# Prerequisites: Signatures are stored in a registry referencing the signed OCI artifact
+notation inspect localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+```
+
+An example output:
```

Comment:
`notation inspect` prints a tree view by default, which usually requires a wide screen.

The screen size of a typical TTY is `80 x 24`. How can the tree view fit the width of `80`?

Reply:
> Will a text view be better for human beings?

Look into text view for the next iteration. Carry over this comment along with the list --verbose item into a separate issue.

Reply:
@vaninrao10 @priteshbandi Could you explain what you are going to do for `list --verbose`? If we are going to use `list --verbose` command to provide the inspect feature, then we could copy the content from `inspect.md` to `list.md`, and drop `notation inspect` command. For the text view, we could do it for next iteration.

Reply:
though "notation list -- verbose" looks right, is not the ideal customer experience in the long term when we want customers to inspect each signature individually. Building a separate inspect command will give good customer experience in our current release and prep us for the future requirements. where the following commands will be needed: 
notation inspect <repo/individual-signature-digest>
notation inspect <FilenameOnLocalDrive>

Reply:
It's acceptable that use `notation list` to list how many signatures are associated with an artifact and use `notation inspect` to look into the details of a specific signature. If we align to keep `notation inspect` in RC.2, can we add an `experimental` label for this new flag?

Reply:
Tracking in a separate issue https://github.com/notaryproject/notation/issues/506 and will be taken iteratively.